install/firmware.csv

```diff
@@ -1328,71 +1328,85 @@ Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50037,4.93,2015-07-23T00:00:00Z,2016-
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50037,5.14,2016-12-13T00:00:01Z,2016-12-20T08:19:43Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50037,5.15,2016-12-20T08:19:44Z,2018-10-23T22:43:36Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50037,5.22,2018-10-23T22:43:37Z,2020-07-14T03:32:19Z,enable GPS-L5
-Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50037,5.45,2020-07-14T03:32:20Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50037,5.45,2020-07-14T03:32:20Z,2020-09-04T00:16:14Z,
+Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50037,5.45,2020-09-04T00:16:15Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50041,4.85,2015-07-23T00:00:00Z,2016-05-05T22:10:34Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50041,5.10,2016-05-05T22:10:35Z,2016-08-14T19:13:34Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50041,5.14,2016-08-14T19:13:35Z,2016-12-19T22:47:07Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50041,5.15,2016-12-19T22:47:08Z,2017-06-19T04:56:38Z,
-Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50041,5.22,2017-06-19T04:56:39Z,9999-01-01T00:00:00Z,enable GPS-L5
+Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50041,5.22,2017-06-19T04:56:39Z,2020-09-04T00:28:32Z,enable GPS-L5
+Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50041,5.45,2020-09-04T00:28:33Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50043,4.93,2015-07-23T00:00:00Z,2016-08-23T01:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50043,5.10,2016-08-23T01:00:01Z,2016-08-24T00:19:57Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50043,5.14,2016-08-24T00:19:58Z,2016-12-20T14:45:52Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50043,5.15,2016-12-20T14:45:53Z,2017-06-20T16:29:04Z,
-Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50043,5.22,2017-06-20T16:29:05Z,9999-01-01T00:00:00Z,enable GPS-L5
+Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50043,5.22,2017-06-20T16:29:05Z,2020-09-07T02:13:55Z,enable GPS-L5
+Trimble Navigation Ltd.,TRIMBLE NETR9,5522R50043,5.45,Z,9999-01-01T00:00:00Z,
```

Comment:
Please revert this line to its original entry (fw 5.22), as I don't think VGPK (rec sn 5522R50043) firmware upgrade went through, receiver is still on 5.22

```
runpkr00:  Version 5.40 (Linux)
runpkr00:  T01Lib: 8.63 
runpkr00:  DAT: VGPK202009072100a.tg!
runpkr00:  Input file: VGPK202009072100a.T02
runpkr00:  records: 245
runpkr00:  status : 0
filename:                VGPK202009072100a.tgd
file format:             Trimble .dat
file size (bytes):       178088
start date & time:       2020-09-07 21:00:00.000
final date & time:       2020-09-07 21:59:30.000
sample interval:         30.0000
possible missing epochs: 0
4-char station code:     VGPK
station name:            VGPK
station ID number:       VGPK    
antenna ID number:       27561917
antenna type:            TRM57971.00     NONE
antenna latitude (deg):     -39.289299532
antenna longitude (deg):    175.346384768
antenna elevation (m):           788.9777
antenna height (m):      0.0550
receiver ID number:      5522R50043
receiver type:           TRIMBLE NETR9
receiver firmware:       5.22
```